dnsaas/urls.py

```diff
@@ -26,6 +29,7 @@
 
 urlpatterns = patterns(
     '',
+    url(r'^admin2/', include(admin_site2.urls)),
```

Comment:
this new admin should be under `admin` url - the old one under `admin-deprecated` or sth